File: ts/wam/3686.ts
export const WamMessageSecretErrors = 3686 as const

export interface MessageSecretErrorsEvent {
  /** field 1, wire `message_secret_allowed_list` */
  messageSecretAllowedList?: MESSAGESECRETALLOWEDTYPE
  /** field 2, wire `message_secret_error` */
  messageSecretError?: MESSAGESECRETERRORTYPE
  /** field 3, wire `message_media_type` */
  messageMediaType?: MEDIATYPE
}

File: rust/wam/3686.rs
pub const WAM_MESSAGE_SECRET_ERRORS: u32 = 3686;

#[derive(Debug, Default)]
pub struct MessageSecretErrorsEvent {
    /// field 1, wire `message_secret_allowed_list`
    pub message_secret_allowed_list: Option<MESSAGESECRETALLOWEDTYPE>,
    /// field 2, wire `message_secret_error`
    pub message_secret_error: Option<MESSAGESECRETERRORTYPE>,
    /// field 3, wire `message_media_type`
    pub message_media_type: Option<MEDIATYPE>,
}